dev/reference/tools/data-migration/dm-portal.md

```diff
@@ -0,0 +1,235 @@
+
+* 如果在本地启动，浏览器访问 `127.0.0.1:8280` 即可使用。
+* 如果在服务器上启动，需要为服务器上配置访问代理。
+
+### DM Ansible 部署
+
+可以使用 DM Ansible 部署 DM Portal，具体部署方法参照 [使用 DM Ansible 部署 DM 集群](/how-to/deploy/data-migration-with-ansible.md)。
+
+## 使用说明
+
+### 新建规则
+
+#### 功能描述
+
+用于新建一个 `task.yaml` 文件，需要选择同步模式、配置上下游实例、配置库表路由，配置 binlog 过滤。
+
+#### 操作步骤
+
+* 登录 dm-portal 页面，点击**新建任务规则**。
+
+### 基础信息配置
+
+#### 功能描述
+
+用于填写任务名称，以及选择任务类型。
+
+#### 前置条件
+
+已选择**新建同步规则**。
+
+#### 操作步骤
+
+* 填写任务名称。
+* 选择任务类型。
+![DM Portal BasicConfig](/media/dm-portal-basicconfig.png)
+
+### 实例信息配置
+
+#### 功能描述
+
+用于配置上下游实例信息，包括 Host、Port、Username、Password。
+
+#### 前置条件
+
+已填写任务名称和选择任务类型。
+
+#### 注意事项
+
+如果任务类型选择**增量**或者**All**，在配置上游实例信息时候，还需要配置 binlog-file 和 binlog-pos。
+
+#### 操作步骤
+
+* 填写上游实例信息。
+* 填写下游实例信息。
+* 点击**下一步**。
+
+![DM Portal InstanceConfig](/media/dm-portal-instanceconfig.png)
+
+### binlog 过滤配置
+
+#### 功能描述
+
+用于配置上游的 binlog 过滤，可以选择需要过滤的 DDL/DML，并且在数据库上配置的 filter 后会自动给其下的数据表继承。
+
+#### 前置条件
+
+已经配置好上下游实例信息并且连接验证没问题。
+
+#### 注意事项
+
+* binlog 过滤配置只能在上游实例处进行修改编辑，一旦数据库或者数据表被移动到下游实例后，就不可以进行修改编辑。
+* 在数据库上配置的 binlog 过滤会自动被其下的数据表继承。
+
+#### 操作步骤
+
+* 点击需要配置的数据库或者数据表。
+* 点击编辑按钮，选择需要过滤的 binlog 类型。
+
+![DM Portal InstanceShow](/media/dm-portal-instanceshow.png)
+
+![DM Portal BinlogFilter 1](/media/dm-portal-binlogfilter-1.png)
+
+![DM Portal BinlogFilter 2](/media/dm-portal-binlogfilter-2.png)
+
+### 库表路由配置
+
+#### 功能描述
+
+可以选择需要同步的数据库和数据表，并且进行修改名称、合并库、合并表等操作。可以对上一步操作进行撤销，可以对库表路由配置进行全部重置。
+
+#### 前置条件
+
+* 已经配置好需要的 binlog 过滤规则。
+
+#### 注意事项
+
+* 在合并库表操作的时候，不允许批量操作，只能一个个拖动。
+* 在合表库表操作的时候，只能对数据表进行拖动操作，不能对数据库进行数据库进行拖动操作。
+
+#### 操作步骤
+
+* 在**上游实例**处，选择需要同步的数据库和数据表。
+* 点击移动按钮，将需要同步的库表移动至**下游实例**处。
+* 点击右键按钮，可以对库表进行改名操作。
+* 选中需要操作的数据表，可以拖动至别的数据表图标上创建出合并表；可以拖动到数据库图标上移动至该库下；可以拖动到 target-instence 图标上移动到一个新的数据库下。
+* 点击**完成**，自动下载 `task.yaml` 到本地，并且在 DM Portal 服务器上的 `/tmp/` 目录下自动创建一份 `task.yaml` 配置文件。
+
+##### 移动同步库表
+
+![DM Portal TableRoute 1](/media/dm-portal-tableroute-1png)
+
+![DM Portal TableRoute 2](/media/dm-portal-tableroute-2.png)
+
+##### 右键修改库表名称
+
+![DM Portal ChangeTableName](/media/dm-portal-changetablename.png)
+
+##### 合并数据表操作
+
+![DM Portal MergeTable 1](/media/dm-portal-mergetable-1.png)
+
+![DM Portal MergeTable 2](/media/dm-portal-mergetable-2.png)
+
+##### 移动数据表至其他数据库
+
+![DM Portal MoveToDB 1](/media/dm-portal-movetodb-1.png)
+
+![DM Portal MoveToDB 2](/media/dm-portal-movetodb-2.png)
+
+##### 移动数据表至新建默认数据库
+
+![DM Portal MoveToNewDB 1](/media/dm-portal-movetonewdb-1.png)
+
+![DM Portal MoveToNewDB 2](/media/dm-portal-movetonewdb-2.png)
+
+##### 撤销本次操作
+
+![DM Portal Revert](/media/dm-portal-revert.png)
+
+##### 清空下游实例
+
+![DM Portal Reset](/media/dm-portal-reset.png)
+
+### 获取配置
+
+#### 功能描述
+
+在完成任务配置后，DM Portal 会帮忙生成对应的 `task.yaml` 文件。
+
+#### 前置条件
+
+已经配置好所有任务配置。
+
+#### 注意事项
+
+* `task.yaml` 配置文件会在浏览器本地下载目录保存一份，同时在服务器端的 `/tmp/` 目录下保存一份。
+
+#### 操作步骤
+
+* 点击**完成并下载**

```

Comment:
Only one step? If yes, it's unnecessary to use a list.

Reply:
Okay, i will merge this step.

Reply:
Update in https://github.com/pingcap/docs-cn/pull/1679/commits/223bb43e38c9b531a072d68187f2c07b9e942f68